### DISCUSSION TITLE
Update 6.9

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#237] [Release]
* **fix Android 12 manifest error**
* updated libraries
* fix rename
* add "show un-tagged" action in tag manager and tags menu
* allow position change of "show un-tagged" action in the tags menu
* add option to choose contacts mime type (some code came from KISS)
* restore settings activity back-stack after changing theme
* fix tag order dialog crash
* fix contact icon cache
* add simple and highlight theme generators #176
* add dial (search) provider
* improve "dock" layout (reuse views)
* fix shortcut icon (dashes were not straight)
* fix crash (NPE) on double touch
* allow custom icons for search results
* fix "Add search engine" not working

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v6.9'>Update 6.9</a>.</em>

**Comment by ltguillaume** [comment]
Looks great, no new issues found 🙂
